frontend/benefit/applicant/src/context/ApplicationContext.tsx

```diff
@@ -16,6 +17,7 @@ const ApplicationContext = React.createContext<ApplicationContextType>({
   },
   isLoading: true,
   setApplicationTempData: () => noop,
+  setCurrentStep: () => noop,
```

Comment:
`setCurrentStep: noop` is correct. 
Imho I would try to avoid lodash package and create the methods myself, I have bad experiences about the size of it and treeshaking not working properly.

Reply:
fixed